region/crateria/central/Landing Site.json

```diff
@@ -632,14 +649,52 @@
           "steepDownTiles": 1,
           "openEnd": 2
         }},
-        {"shinespark": {"frames": 95, "excessFrames": 33}}
+        {"or": [
+          {"shinespark": {"frames": 95, "excessFrames": 33}},
+          {"and": [
+            "canShinechargeMovementTricky",
+            {"shinespark": {"frames": 87, "excessFrames": 33}}
+          ]}
+        ]}
       ],
       "clearsObstacles": ["A"],
       "note": [
         "Store the spark on the right side of the ledge.  Run left and do a big jump towards the gauntlet door and then midair spark at the right time.",
         "The lines in the background make for a good way to measure height."
       ]
     },
+    {
+      "link": [4, 1],
+      "name": "Big Jump Midair Shinespark Through Door",
+      "requires": [
+        "canMidairShinespark",
+        "canShinechargeMovementComplex",
+        {"canShineCharge": {
+          "usedTiles": 19,
+          "steepUpTiles": 2,
+          "steepDownTiles": 1,
+          "openEnd": 2
+        }},
+        {"or": [
+          {"shinespark": {"frames": 96}},
+          {"and": [
+            "canShinechargeMovementTricky",
+            {"shinespark": {"frames": 88}}
+          ]}
+        ]}
+      ],
+      "exitCondition": {
+        "leaveWithSpark": {}
+      },
+      "unlocksDoors": [{"types": ["ammo"], "requires": []}],
+      "note": [
+        "Store the spark on the right side of the ledge.  Run left and do a big jump towards the gauntlet door and then midair spark at the right time.",
+        "The lines in the background make for a good way to measure height."
+      ],
+      "devNote": [
+        "Unlocking the door is free since the obstacle C cleared means that any lock was already taken care of."
```

Comment:
Good catch, it definitely should have "C" as a req. If possible it seems better to avoid implicitly assuming a refill is available at the Ship, in case the Ship behavior were to be modified.